src/components/Editor/Preview/index.js

```diff
@@ -70,7 +70,7 @@ class Preview extends PureComponent {
   onMouseOver(e) {
     const { target } = e;
     if (this.props.selectedFrameVisible) {
-      this.props.updatePreview(target, this.props.editor);
+      updatePreview(target, this.props.editor, this.props);
```

Comment:
looks like this needs a rebase :)

Reply:
This is after a rebase with the newest upstream at the time of pushing :/

Reply:
fixed it manually.